pycroft/model/config.py

```diff
@@ -3,6 +3,8 @@
 # the Apache License, Version 2.0. See the LICENSE file for details.
 from sqlalchemy import CheckConstraint, Column, ForeignKey, Integer
 from sqlalchemy.orm import relationship
+
+from pycroft.model.address import Address
```

Comment:
Probably unnecessary import

Reply:
Good catch, there was one in the `ConfigFactory` as well.